### ISSUE TITLE
Apt-get issue after plex install

### ISSUE THREAD

**Issue by karl0ss** [htpcBeginner/AtoMiC-ToolKit#178]
I have installed plex, and plex is running and working fine, but whenever I perform an apt-get command I get the following output..

```
Setting up plexmediaserver (1.5.6.3790-4613ce077) ...
OK
Failed to execute operation: File exists
dpkg: error processing package plexmediaserver (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 plexmediaserver
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Any ideas?

**Comment by ZaLiTHkA** [comment]
Just chiming in to say: I've been getting the same error lately.. Running Ubuntu 16.04.2 LTS.

**Comment by Anderskars** [comment]
same issue, running 16.04 LTS (in hyper-v)

**Comment by TommyE123** [comment]
Just to let everyone know i'm working on a fix for this which is proving slightly tricky.
Thanks to everyone for letting me know its not working.
I'll come back to you with a fix as quick as I can.

Tom

**Comment by karl0ss** [comment]
Thanks @TommyE123 it doesn't "break" anything, just annoying...

**Comment by tazboyz16** [comment]
Try running "sudo dpkg --configure -a" or "sudo apt-get -f install" should fix up that kind of error for it repeating is due to a broken deps but has previously installed version

**Comment by TommyE123** [comment]
Hi,

I’ve pushed up some changes that I hope has fixed the issues around this and would be grateful if you could give it another try and report back with your results?

Thanks
Tom

**Comment by ZaLiTHkA** [comment]
@TommyE123, do we need to reinstall PMS in order to check this for you?

**Comment by TommyE123** [comment]
Due the nature of the errors I think the only sure fire way to fix the issues is to remove and reinstall i'm afraid. This is because the source file needs to be the file Plex installed as part of the package and then I've just edited it after to enable it. (I create a temp file first just to install it which gets overwritten)
I'm hoping that this is the last of the issues but until they release a further update and check it updates ok I can't be 100%.

Sorry it looks like its got to be this way but I can't see another way round it at the minute.

Tom

**Comment by ZaLiTHkA** [comment]
Yeah, that all makes sense.. I just wanted to double check first. :)

Anyways, here's an interesting one; after making a backup of my PMS installation, I uninstalled Plex through this toolkit, which seemed to go through without a hitch. But when I try reinstall, I see the following message in the middle of the script process:

```
--->Installing Plex Media Server...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package plexmediaserver
```

As per this message, I don't see `plexmediaserver` in my list of services and I can't access the web interface. I've copied the full output of the script and [saved it in a gist](https://gist.github.com/ZaLiTHkA/470d193462228d2a8c9bd7c21e1f8abe) for reference, not sure if there's anything else useful there though.

Perhaps I should've asked this first, but which branch of this repo should I be using for this? It looks like the changes are straight on `master`, as the so-called `Dev` branch is far behind it. Is this correct?
